guide/how-to-test-your-model-mod-routines.rst

```diff
@@ -1,11 +1,14 @@
 How to test your model_mod routines
 ===================================
 
-The program ``model_mod_check.f90`` can be used to test the routines
-individually before running them with *filter*. Add a ``mkmf_model_mod_check``
-and ``path_names_model_mod_check`` to your ``DART/models/your_model/work``
-subdirectory. You might find it helpful to consult another model matching your
-model type (simple or complex). See the documentation for :doc:`model_mod_check 
-<../assimilation_code/programs/model_mod_check/model_mod_check>` in
-``DART/assimilation_code/programs/model_mod_check`` for more information on the
-tests available.
+The program ``model_mod_check.f90`` can be used to test model_mod routines
+individually before running them with filter. Add ``model_mod_check``
+to the list of programs in ``DART/models/your_model/work/quickbuid.sh`` to
+build ``model_mod_check`` with ``quickbuild.sh``.
+
+For more information on the tests in ``model_mod_check`` see :doc:`model_mod_check 
```

Comment:
Is there a functional difference here between using :doc:`model_mod_check and :ref:`model_mod_check`?

Reply:
I just looked this up, 
:ref: is to for explicit targets
:doc: is for a page (so relative location)

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

Going forwards, I think :ref: is better because you can move the files around however you like (this would break a :doc: link).